### PR TITLE
Fixes in Monad comprehensions doc

### PR DESCRIPTION
**Pull request by pakoito**
Fix one example as reported on Gitter

### CODE REVIEW

File: modules/docs/arrow-docs/docs/docs/arrow-core/patterns/monadcomprehensions/README.md [hunk 1]
```diff
@@ -188,8 +188,8 @@ For this purpose, the typeclass [`MonadError`]({{ '/docs/arrow/typeclasses/monad
 [`MonadError`]({{ '/docs/arrow/typeclasses/monaderror' | relative_url }}) allows us to raise and recover from errors.
 
 ```kotlin
-fun getLineLengthAverage(path: FilePath): IO<List<String>> =
-  IO.fx {
+fun <F> MonadError<F, Throwable>.getLineLengthAverage(path: FilePath): Kind<F, List<String>> =
```
Comment (1Jajen1) [inline modules/docs/arrow-docs/docs/docs/arrow-core/patterns/monadcomprehensions/README.md]: The code below supposedly reads from a file which should at least be `MonadDefer` I think. I don't think it matters in an example for `monadError` comprehensions but maybe an example without side-effects might be better.

Comment (pakoito) [reply modules/docs/arrow-docs/docs/docs/arrow-core/patterns/monadcomprehensions/README.md]: I've added async in the middle

File: modules/docs/arrow-docs/docs/docs/arrow-core/patterns/monadcomprehensions/README.md [hunk 2]
```diff
@@ -185,12 +185,14 @@ This exception goes uncaught and finalizes the program with a crash. Knowing thi
...
-[`MonadError`]({{ '/docs/arrow/typeclasses/monaderror' | relative_url }}) allows us to raise and recover from errors.
+It allows us to raise and recover from errors.
+
+The typeclass [`Async`]({{ '/docs/apidocs/arrow-fx/arrow.fx.typeclasses/-async/index.html' | relative_url }}) allows us to wrap effectful operations, and it inherits all the operators from [`MonadError`]({{ '/docs/arrow/typeclasses/monaderror' | relative_url }}).
 
 ```kotlin
-fun getLineLengthAverage(path: FilePath): IO<List<String>> =
-  IO.fx {
-    val (file) = getFile(path)
+fun <F> Async<F, Throwable>.getLineLengthAverage(path: FilePath): Kind<F, List<String>> =
```
Comment (1Jajen1) [inline modules/docs/arrow-docs/docs/docs/arrow-core/patterns/monadcomprehensions/README.md]:
```suggestion
fun <F> Async<F>.getLineLengthAverage(path: FilePath): Kind<F, List<String>> =
```
Can we run ank on these?

Comment (pakoito) [reply modules/docs/arrow-docs/docs/docs/arrow-core/patterns/monadcomprehensions/README.md]: @nomisRev is the final design with our without it?

Comment (1Jajen1) [reply modules/docs/arrow-docs/docs/docs/arrow-core/patterns/monadcomprehensions/README.md]: Future proofing for bifunctor io?

Comment (pakoito) [reply modules/docs/arrow-docs/docs/docs/arrow-core/patterns/monadcomprehensions/README.md]: hahaha I didn't refresh the page before merging

**Pull request merged**